src/components/authPage/LoginPage.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from "react";
+import { NavLink } from "react-router-dom";
+import { useAuth } from "../../context/authContext/AuthProvider";
+
+const emailMatchPattern =
```

Comment:
Any specific reason you are adding this ? The HTMl tag which you use for email already does an inbuilt checking for correct format.

Reply:
Actually I am not submitting the form and just manually checking. So that's the reason I have use pattern matching.

Reply:
Makes sense, got it!